autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/configuration/RuntimeConfigurationWriter.kt

```diff
@@ -77,10 +77,12 @@ class RealRuntimeConfigurationWriter @Inject constructor(val moshi: Moshi) : Run
      */
     override fun generateUserUnprotectedDomains(): String {
         return """
-            userUnprotectedDomains = [];
+            "userUnprotectedDomains" : []
         """.trimIndent()
     }
 
+    // TODO: we need to put back "unknown_username_categorization": $unknownUsernameCategorization" into JSON.
```

Comment:
MUST DO before merging PR into develop

Reply:
I enabled it locally, and autofill seems to work fine!